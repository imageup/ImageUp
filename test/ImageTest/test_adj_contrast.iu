function main() -> int
{
        image: a = read("./images/face1.jpg");
        tuple: x = get_pixel(a, (0,1));
        prints("Before: " + StrParsef(x[0]) + " " + StrParsef(x[1]) + " " + StrParsef(x[2]));
        image: output;
        output = adjust_contrast(a, 1.1);	
        tuple: y = get_pixel(output, (0,1));
        prints("After: " + StrParsef(y[0]) + " " + StrParsef(y[1]) + " " + StrParsef(y[2]));    
	    save("test_contrast.out.jpg", output);

        return 0;
}
