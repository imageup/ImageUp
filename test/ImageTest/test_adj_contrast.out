Before: 233.000 239.000 238.000
After: 255.000 255.000 255.000
